### DISCUSSION TITLE
PCL莫名自己删除怎么办啊

### DISCUSSION THREAD

**Discussion by rusherwww** [Meloong-Git/PCL#913] [提问] [answered]
之前还好些，偶尔，现在频繁发生，更新最新版就会自己消失，火绒自定义保护也没用，没有任何提示，这该咋办啊

**Comment by WorldHim** [comment]
用的是什么版本？可能被杀软删了？建议看看 Defender 的日志。

**Reply by rusherwww** [reply]
> 用的是什么版本？可能被杀软删了？建议看看 Defender 的日志。

最新的测试版，Defender没有任何提示

**Comment by xuzehao2012** [comment]
有可能被杀软，干掉了...

**Comment by xuzehao2012** [comment]
你试试反馈

**Comment by LTCatt** [comment, marked as answer]
参考 #666 ，现在 Windows Defender 会在没有任何提示和记录的情况下静默删除文件（只能使用行为监测工具捕捉到）……

**Reply by rusherwww** [reply]
> 参考 #666 ，现在 Windows Defender 会在没有任何提示和记录的情况下静默删除文件（只能使用行为监测工具捕捉到）......

关键是我用了火绒啊……

**Reply by rusherwww** [reply]
现在似乎好了

**Reply by Louis-Harsune** [reply]
> 现在似乎好了

因为每个快照版的MD5不一样，所以没法一个个向微软上传误报，正式版因为MD5不变，因此都上传了误报，理论上不会出现此问题。

**Reply by rusherwww** [reply]
> > 现在似乎好了
> 
> 因为每个快照版的MD5不一样，所以没法一个个向微软上传误报，正式版因为MD5不变，因此都上传了误报，理论上不会出现此问题。

顺便问下，停用了安全中心还会有这种情况吗

**Reply by WorldHim** [reply]
> > > 现在似乎好了
> > 
> > 
> > 因为每个快照版的MD5不一样，所以没法一个个向微软上传误报，正式版因为MD5不变，因此都上传了误报，理论上不会出现此问题。
> 
> 顺便问下，停用了安全中心还会有这种情况吗

如果是安全中心导致的，那么停用之后就不会有问题了
